pyvo/io/uws/tree.py

```diff
@@ -278,6 +278,7 @@ class Parameters(UWSElement, HomogeneousList):
     Parameters element of a job
     """
     def __init__(self, config=None, pos=None, _name='parameters', **kwargs):
+        """ """
```

Comment:
I would add a line below to state that the empty `pydoc` is required and should not be removed. Otherwise someone might be tempted to get rid of it and they'll scratch their heads trying to figure out why sphinx is unhappy. I can picture myself in that situation :-).